app/styles/_media_queries.scss

```diff
@@ -320,11 +320,17 @@ only screen and (orientation:landscape) and (min-width:481px) and (max-height:48
     background-image: image-url('spinnerwhite@2x.png');
   }
 
-  .input-row select {
+  .input-row select,
+  .select-row,
+  .select-row.disabled:hover {
     background-image: image-url('ddarrow_inactive@2x.png');
   }
 
-  .input-row select:focus {
```

Comment:
We only have 3 "select" elements, all inside of "select-row"s. I believe we can safely remove the ".input-row select"s.